[2025-01-17T20:38:22.017+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-17T20:38:22.033+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.extract_data manual__2025-01-17T20:38:19.231103+00:00 [queued]>
[2025-01-17T20:38:22.041+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.extract_data manual__2025-01-17T20:38:19.231103+00:00 [queued]>
[2025-01-17T20:38:22.042+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-17T20:38:22.057+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_data> on 2025-01-17 20:38:19.231103+00:00
[2025-01-17T20:38:22.061+0000] {standard_task_runner.py:72} INFO - Started process 115 to run task
[2025-01-17T20:38:22.064+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'extract_data', 'manual__2025-01-17T20:38:19.231103+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp4adbqk_r']
[2025-01-17T20:38:22.066+0000] {standard_task_runner.py:105} INFO - Job 22: Subtask extract_data
[2025-01-17T20:38:22.083+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-01-17T20:38:22.119+0000] {task_command.py:467} INFO - Running <TaskInstance: first_dag.extract_data manual__2025-01-17T20:38:19.231103+00:00 [running]> on host c38648c89b2b
[2025-01-17T20:38:22.195+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-17T20:38:19.231103+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-17T20:38:19.231103+00:00'
[2025-01-17T20:38:22.196+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-17T20:38:22.571+0000] {logging_mixin.py:190} INFO -    Row ID        Order ID  Order Date  ... Quantity Discount    Profit
0       1  CA-2016-152156   11/8/2016  ...        2     0.00   41.9136
1       2  CA-2016-152156   11/8/2016  ...        3     0.00  219.5820
2       3  CA-2016-138688   6/12/2016  ...        2     0.00    6.8714
3       4  US-2015-108966  10/11/2015  ...        5     0.45 -383.0310
4       5  US-2015-108966  10/11/2015  ...        2     0.20    2.5164

[5 rows x 21 columns]
[2025-01-17T20:38:22.573+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-17T20:38:22.582+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-17T20:38:22.582+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=extract_data, run_id=manual__2025-01-17T20:38:19.231103+00:00, execution_date=20250117T203819, start_date=20250117T203822, end_date=20250117T203822
[2025-01-17T20:38:22.599+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-17T20:38:22.617+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-17T20:38:22.619+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
